src/component/TodoApp/TodoApp.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo/TodoInfo';
+
+interface TodoAppProps {
+  todos: Todo[];
+  deleteTodo: (id: number) => void;
+  tempTodo: Todo | null;
```

Comment:
The 'tempTodo' prop seems to be unnecessary. You can remove it and handle the temporary Todo state within the parent component.